CHANGELOG.md

```diff
@@ -21,6 +19,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
 
+## dynamic_bonding_curve [0.1.8] [PR #148](https://github.com/MeteoraAg/dynamic-bonding-curve/pull/148)
+
+### Added
+
+- `create_config_for_dammv2_migration` endpoint to allow create `PoolConfig` account for only DAMM v2 migration with LP vesting configuration.
+- `PoolConfig` account now stores `creator_lp_vesting_info` and `partner_lp_vesting_info` fields. Only applicable to DAMM v2 migration option. It store vesting parameters required for `lock_position` cpi during DAMM v2 migration.
+
+### Breaking Changes
```

Comment:
Could you update change for breaking change that: 
- Endpoints: `create_config` `create_config_for_dammv2_migration` `initialize_virtual_pool_with_spl_token` and `initialize_virtual_pool_with_token2022` will only allow config that has minimum 10% of locked liquidity in at least 1 day?

Reply:
will create PR for that